tests/core/policies/test_rule_policy.py

```diff
@@ -613,7 +642,7 @@ async def test_form_unhappy_path_no_validation_from_rule():
         is_rule_tracker=True,
     )
 
-    policy = RulePolicy()
+    policy = RulePolicy(restrict_rules=False)
```

Comment:
this the only test which actually requires `restrict_rules=False`

Reply:
yes, but I just set it to False in all our tests, so that if we change restriction, it'll not break other tests that don't test for it

Reply:
to separate the test logics